src/commands/copyRelativePathToClipboard.ts

```diff
@@ -0,0 +1,26 @@
+import { env, TextEditor, Uri } from 'vscode';
+import { Commands } from '../constants';
+import type { Container } from '../container';
+import { command } from '../system/command';
+import { ActiveEditorCommand, getCommandUri } from './base';
+
+@command()
+export class CopyRelativePathToClipboardCommand extends ActiveEditorCommand {
+	constructor(private readonly container: Container) {
+		super(Commands.CopyRelativePathToClipboard);
+	}
+
```

Comment:
@d13 To ensure this copies the correct path when you right-click on a commit/etc, we need to handle the `preExecute` call, similar to
https://github.com/gitkraken/vscode-gitlens/blob/79f0d0b2971e506e7fbf5cbaa928288a07e823b7/src/commands/copyShaToClipboard.ts#L28-L50

Without that, we will be trying to use the repository for the open editor, rather than the context provided.

Reply:
When testing in the different sidebar views, `isCommandContextViewNodeHasFileCommit` covered each scenario.